src/PrestaShopBundle/Form/Validator/Constraints/TinyMceMaxLengthValidator.php

```diff
@@ -33,17 +34,17 @@ class TinyMceMaxLengthValidator extends ConstraintValidator
 {
     public function validate($value, Constraint $constraint)
     {
-        $replaceArray = array(
+        $replaceArray = [
             "\n",
             "\r",
             "\n\r",
```

Comment:
I just like how this line is useless when the first and second one can do the job --' 
More I found a potential problem in the future here:

```php
$str     = "Line 1\nLine 2\rLine 3\r\nLine 4\n";
$order   = array("\n", "\r");
$replace = '<br />';

$newstr = str_replace($order, $replace, $str);
var_dump($newstr);

// Will produce
/*
$ php test.php 
/home/got/test.php:10:
string(54) "Line 1<br />Line 2<br />Line 3<br /><br />Line 4<br />"
*/
```

When 
When
```php
$newstr = str_replace($order, [$replace], $str);

// Will produce
/*
$ php test.php 
/home/got/test.php:10:
string(42) "Line 1<br />Line 2Line 3<br />Line 4<br />"*/
```
As you can see, it is replaced by `null` and not an empty string (because index isn't found).

Currently null = '' but this isn't what we expect.

Reply:
we agree I shouldn't change this behavior in this pull request, am I right?

Reply:
I think you can, otherwise we'll never fixed it! (or not now at least) 
Leave your code better than you found it. (https://deviq.com/boy-scout-rule/ in Agile XP)

Reply:
Don't you think it's already far better? My fix concerns only 10 lines of this pull request